tests/testthat/test-keypair.R

```diff
@@ -61,3 +61,34 @@ test_that("exchange_jwt_for_token handles errors correctly", {
     error = TRUE
   )
 })
+
+test_that("JWT generation works with encrypted private key and passphrase", {
+  jwt <- generate_jwt(
+    "account",
+    "user",
+    test_path("encrypted_rsa_key.p8"),
```

Comment:
Consider adding a comment documenting how you created the key (applies to this and the existing keys).